packages/wonder-blocks-modal/docs.md

```diff
@@ -0,0 +1,35 @@
+# Custom modals
+
+```js
+const React = require("react");
+
+const {Title} = require("@khanacademy/wonder-blocks-typography");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+class CustomModal extends React.Component {
```

Comment:
Hmm - I think I have a different definition of a "Custom Modal" than you. I was imagining that a custom one would be entirely built up from the primitive components. I think calling this a "Wrapped Modal" might be more accurate.